libraries/c_sdk/standard/ble/include/iot_ble_mqtt_serialize.h

```diff
@@ -32,8 +32,8 @@
 #ifndef IOT_BLE_MQTT_SERIALIZE_H
 #define IOT_BLE_MQTT_SERIALIZE_H
 
-#include "types/iot_mqtt_types.h"
 #include "aws_clientcredential.h"
+#include "platform/iot_network.h"
```

Comment:
`IotBleMqtt_GetPacketType` and `IotBleMqtt_GetRemainingLength` depends on the old network interface struct. So you can get rid of this header file after refactoring those function definitions.